recipes/harminv/meta.yaml

```diff
@@ -0,0 +1,42 @@
+{% set name = "harminv" %}
+{% set version = "1.4.1" %}
+{% set sha256 = "e1b923c508a565f230aac04e3feea23b888b47d8e19b08816a97ee4444233670" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  fn: {{ name }}-{{ version }}.tar.gz
+  url: https://github.com/stevengj/{{ name }}/releases/download/v{{ version }}/{{ name }}-{{ version }}.tar.gz
+  sha256: {{ sha256 }}
+
+build:
+  number: 0
+  skip: True  # [win]
+
+requirements:
+  build:
+    - toolchain
+    - gcc
+    - libgfortran  # [linux]
+    - openblas
+  run:
+    - libopenblas
+    - libgfortran
+
+test:
+  source_files:
+    - sines
```

Comment:
This doesn't seem to have a real test.  At the very least, could you add an existence test please https://conda-forge.org/docs/testing.html#simple-existence-tests